include/onnxruntime/core/framework/run_options.h

```diff
@@ -32,6 +32,8 @@ struct OrtRunOptions {
   bool training_mode = true;
 #endif
 
+  bool capture_cuda_graph = false;
```

Comment:
Please see /include/onnxruntime/core/session/onnxruntime_run_options_config_keys.h to see how to add a new RunOption (it will appear in the config_options field below)

Reply:
Thanks. Will change it.

Reply:
RunOption may not be needed based on the discussion here: https://github.com/microsoft/onnxruntime/pull/10423#discussion_r797214220

Reply:
Ok. We can add it in the future if needed.